cmd/dmsg-server/commands/root.go

```diff
@@ -92,7 +93,7 @@ var RootCmd = &cobra.Command{
 			MaxSessions:    conf.MaxSessions,
 			UpdateInterval: conf.UpdateInterval,
 		}
-		srv := dmsg.NewServer(conf.PubKey, conf.SecKey, disc.NewHTTP(conf.Discovery), &srvConf, m)
+		srv := dmsg.NewServer(conf.PubKey, conf.SecKey, disc.NewHTTP(conf.Discovery, http.Client{}), &srvConf, m)
```

Comment:
What are we doing with this http client?

Reply:
Previously we were using an empty HTTP client for the connection [here](https://github.com/skycoin/dmsg/blob/7f22ff65ff0dd7ae00019a090192e3c2572457af/disc/client.go#L46). To accommodate the dmsg-http client I had to make a few changes so we now have to provide a `http.Client` to `dmsg.NewServer`.